force-app/main/default/flows/Send_Email.flow-meta.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
flow entry condition is not correct, how is it possible to compare date with date/time.